src/qat/purr/backends/qiskit_simulator.py

```diff
@@ -247,14 +281,40 @@ def execute(self, builder: QiskitBuilder) -> Dict[str, Any]:
             AerSimulator._DEFAULT_CONFIGURATION | {"open_pulse": False}
         )
         aer_config.n_qubits = self.model.qubit_count
-        qasm_sim = AerSimulator(aer_config, noise_model=builder.model.noise_model)
+
+        # AerSimulator can call a range of backends. We allow these to be choosen, along with
+        # any relevant arguments.
+        config = self.create_config(kwargs)
```

Comment:
custom options could probably come from qat config ... some section that says: simulation -> backend -> method, ...

Reply:
Done: for now, just done simulation -> method. Happy to revisit further branching if this becomes more extensive :smile: